cfme/common/vm.py

```diff
@@ -40,6 +43,11 @@
     ('cancel_button', form_buttons.cancel)
 ])
 
+drift_table = CheckboxTable({
```

Comment:
We've been dropping version picking <5.6, can likely drop this and just use the latest locator.